workspaces/liferay-customer-workspace/client-extensions/liferay-customer-etc-spring-boot/src/main/java/com/liferay/customer/service/UserAccountService.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.customer.service;
+
+import com.liferay.client.extension.util.spring.boot.BaseRestController;
+
+import java.util.ArrayList;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.oauth2.jwt.Jwt;
+import org.springframework.stereotype.Component;
+
+/**
+ * @author Felipe Franca
+ */
+@Component
+public class UserAccountService extends BaseRestController {
+
+	public ArrayList<String> getUserRoles(Jwt jwt) {
+		JSONObject userJSONObject = _getMyUserAccountJSONObject(jwt);
+
+		JSONArray roleBriefsJSONArray = userJSONObject.getJSONArray(
+			"roleBriefs");
+
+		ArrayList<String> rolesNames = new ArrayList<>();
```

Comment:
Use generic objects (ie list) to pass around unless you need a specific ArrayList method

Reply:
Done!